xcode/Subconscious/Shared/Components/Common/Suggestion/SuggestionLabelView.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  SuggestionLabelView.swift
+//  Subconscious
+//
+//  Created by Gordon Brander on 1/28/22.
+//
+
+import SwiftUI
+
+struct SuggestionCreateHeading: View {
+    var fallback: String
+
+    private var preview: String {
+        fallback.trimmingCharacters(in: .whitespacesAndNewlines)
+    }
+
+    var body: some View {
+        if !preview.isEmpty {
+            HStack {
+                Text("New note")
+                Text(verbatim: #""\#(preview)""#)
+                    .foregroundColor(.secondary)
+            }
+        } else {
+            HStack {
+                Text("New note")
+            }
+        }
+    }
+}
+
+struct SuggestionLabelView: View, Equatable {
+    var suggestion: Suggestion
+    var empty = String(localized: "Empty note")
+
+    var body: some View {
+        switch suggestion {
+        case let .memo(address, title):
+            Label(
+                title: {
+                    TitleGroupView(
+                        title: Text(
+                            verbatim: !title.isEmpty ? title : empty
```

Comment:
nitpick: another double negative

Reply:
These come down to me liking to read the "default" case first and the corner case second. I can see your POV on double negative though.